slsDetectorSoftware/src/CmdApp.cpp

```diff
@@ -9,15 +9,16 @@ int main(int argc, char *argv[]){
 
     sls::CmdParser parser;
     parser.Parse(argc, argv);
-
-    std::cout << "cmd: " << parser.command() << "\narguments: ";
-    for (auto& arg : parser.arguments()){
-        std::cout << arg << " ";
-    }
-    std::cout << "\n\n";
     
     int default_action = -1;
     sls::Detector d(parser.multi_id());
     sls::Caller c(&d);
+
```

Comment:
This now calls both get and put which might be convenient for testing but has the problem that we need different number of arguments